keepalived/check/check_smtp.c

```diff
@@ -82,6 +82,37 @@ dump_smtp_check(void *data)
 	dump_list(smtp_checker->host);
 }
 
+static int
```

Comment:
I prefer type bool, return true for same (currently 0), and false if different and change goto err to return false